packages/p2p/src/components/order-details/order-details-footer.jsx

```diff
@@ -112,7 +112,7 @@ const OrderDetailsFooter = observer(() => {
         return (
             <React.Fragment>
                 <div className='order-details-card__footer'>
-                    <div className='order-details-card__footer--left'>
+                    <div className='order-details-card__footer--right'>
```

Comment:
Are we still using the `--left` class? If not, please remove it

Reply:
Nope, there are no more components using `order-details-card__footer--left`, ill remove the class asap